llvm/lib/Bitcode/Reader/MetadataLoader.cpp

```diff
@@ -731,6 +731,29 @@ class MetadataLoader::MetadataLoaderImpl {
     upgradeCULocals();
   }
 
+  void cloneLocalTypes() {
+    for (int I = MetadataList.size() - 1; I >= 0; --I) {
```

Comment:
any particular reason this iterates in reverse? Probably worth a comment explaining it?

Reply:
Sorry, it's just a residue from the function draft. Fixed that.